crates/node/src/geth.rs

```diff
@@ -96,9 +98,19 @@ impl Instance {
 
     /// Spawn the go-ethereum node child process.
     ///
-    /// [Instance::init] must be called priorly.
+    /// [Instance::init] must be called prior.
+    #[tracing::instrument(skip_all, fields(geth_node_id = self.id))]
     fn spawn_process(&mut self) -> anyhow::Result<&mut Self> {
-        self.handle = Command::new(&self.geth)
+        let node_logs_file_path = self.base_directory.join("logs").join("node.log");
```

Comment:
We spawn multiple nodes to allow parallel test execution (there will be many tests). I think the log file should include the node ID or some other identifier.